env/env.postgres

```diff
@@ -1,5 +1,5 @@
 POSTGRES_USER=admin
 POSTGRES_PASSWORD=password
-POSTGRES_DB=postgres_db
+POSTGRES_DB=agent0_db
```

Comment:
nit: if we expect to have more than one db present on the postgres instance, we might want to indicate that by changing the env variables in the `.env` file.... for example

```sh
# .env
AGENT0_POSTGRES_USER=admin
AGENT0_POSTGRES_PASSWORD=password
AGENT0_POSTGRES_DB=agent0_db
MY_POSTGRES_USER=me
MY_POSTGRES_PASSWORD=yo
MY_POSTGRES_DB=my_db
```

then in the compose file....
```yaml
environment:
      POSTGRES_PASSWORD: $AGENT0_POSTGRES_PASSWORD
      POSTGRES_USER: $AGENT0_POSTGRES_USER
      POSTGRES_DB: $AGENT0_POSTGRES_DB
```

might need to pass as args rather than env vars since I'm not sure if `.env` vars will be interpolated but hopefully the gist makes sense